mk/spksrc.cross-dotnet-env.mk

```diff
@@ -0,0 +1,82 @@
+# Configuration for dotnet build
+# 
+
+# NOTE: 32bit (x86) is not supported:
+# https://github.com/dotnet/core/issues/5403
+# https://github.com/dotnet/core/issues/4595
+GENERIC_ARCHS = ARM7
+UNSUPPORTED_ARCHS += $(PPC_ARCHES) $(ARM5_ARCHES) $(x86_ARCHES)
+
+DOTNET_OS = linux
+
+# Define DOTNET_ARCH for compiler
+ifeq ($(findstring $(ARCH),$(ARM7_ARCHES)),$(ARCH))
+	DOTNET_ARCH = arm
+endif
+ifeq ($(findstring $(ARCH),$(ARM8_ARCHES)),$(ARCH))
+	DOTNET_ARCH = arm64
+endif
+ifeq ($(findstring $(ARCH),$(x86_ARCHES)),$(ARCH))
+	DOTNET_ARCH = x86
+endif
+ifeq ($(findstring $(ARCH),$(x64_ARCHES)),$(ARCH))
+	DOTNET_ARCH = x64
+endif
+ifeq ($(DOTNET_ARCH),)
+	# don't report error to use regular UNSUPPORTED_ARCHS logging
+	$(warning Unsupported ARCH $(ARCH))
+endif
```

Comment:
This is outdated
- GENERIC_ARCHS is removed from framework
- PPC_ARCHES is now PPC_ARCHS
- ARM5_ARCHES is now ARMv5_ARCHS
- etc.